frontend/src/pages/methodology.tsx

```diff
@@ -181,6 +181,20 @@ const Methodology = () => {
             We currently rely on data labelled manually by domain experts; this limits the pace at which it is possible for new data to be incorporated into our dataset. We are
             developing the capacity to automatically label data using machine learning models, to increase the efficiency, sophistication and thoroughness of our data labelling.
           </p>
+          <h2>Natural language search</h2>
+          <p>
+          Natural language search allows you to find what you’re looking for without having to type out precise keywords. This is useful because often certain concepts are described in lots of different ways - like internal combustion engines, internal combustion engine vehicle, ICEV, fossil fuel car, and gasoline car all describe most of the cars you’ll find on today’s roads. With natural language search, you can use the terms you use and hear everyday because the tool will recognise them as related and relevant terms, meaning you get a much richer search experience as a result.
+          </p>
+          <p>
+          We use a machine learning method called dense retrieval alongside a fuzzy string search to perform natural language search. You can find more details <a href="https://climatepolicyradar.org/latest/building-natural-language-search-for-climate-change-laws-and-policies">in our blog post</a>.
```

Comment:
Please add the attribute `target="_blank"` to all these hyperlinks to ensure they open in a new tab.

Reply:
Do you want me to do it for all links in the page, not just the ones I've just added? The existing links didn't have `target="_blank"` so I just copied that

Reply:
done for all links on the page